src/application.py

```diff
@@ -489,13 +489,18 @@ def contest(contest_id):
     info = db.execute(("SELECT * FROM contest_problems WHERE contest_id=:cid "
                        "AND draft=0 ORDER BY problem_id ASC, category ASC"),
                       cid=contest_id)
+
     for row in info:
+        sols = db.execute("SELECT COUNT(DISTINCT user_id) FROM contest_solved WHERE problem_id=:problem_id", problem_id=row["problem_id"])[0]["COUNT(DISTINCT user_id)"]
```

Comment:
Should be `WHERE problem_id=:problem_id AND contest_id=:contest_id` since there could be multiple contests with the same problem ID